### PR TITLE
New thoughts

### PR DESCRIPTION
**Pull request by mcclowes**

### CODE REVIEW

**Comment by mcclowes** [comment]
Lol bad time to choose to use prettier

File: index.js [hunk 1]
```diff
@@ -31,44 +32,82 @@ const preamble = [
...
-	"lovely ladies",
-	"lovely lovely ladies",
-	"lovely lovely lovely ladies",
-	"lovely lovely lovely lovely ladies",
-	"of the fairer sex",
-	"senioritas",
-	"skirts",
-	"spicy mamma jammas",
-	"womenfolk",
+	"behind keyboards",
```
Comment (FreddieRidell) [inline index.js]: I think the removal of the sexism may be a bug not a feature, in this case

Comment (mcclowes) [reply index.js]: fixed

**Review by FreddieRidell — APPROVED** [review]
LGTM

**Pull request merged**